common/Jackdmp.cpp

```diff
@@ -202,6 +202,7 @@ static void usage(FILE* file, jackctl_server_t *server, bool full = true)
             "               [ --port-max OR -p maximum-number-of-ports]\n"
             "               [ --slave-backend OR -X slave-backend-name ]\n"
             "               [ --internal-client OR -I internal-client-name ]\n"
+            "               [ -C configuration-file ]\n"
```

Comment:
Configuration is really a bad name here.
I'm not a native english speaker, so thinking of names is something I know I'm bad at...